File: worker/src/check_db.js
require('dotenv').config({ path: '../.env' });
const { PrismaClient } = require('@prisma/client');
const prisma = new PrismaClient();

async function main() {
    const company = await prisma.company.findUnique({
        where: { ico: "51078856" },
        include: { sources: true }
    });
    if (!company) { console.log("Company not found"); return; }
    for (const s of company.sources) {
        if (s.sourceType === "FS_DANOVE_SUBJEKTY") {
            console.log(s.findings);
        }
    }
}

main()
  .catch(e => console.error(e))
  .finally(async () => {
    await prisma.$disconnect();
  });

File: worker/src/check_db.py
import asyncio
from prisma import Prisma

async def main():
    db = Prisma()
    await db.connect()
    company = await db.company.find_unique(where={"ico": "51078856"}, include={"sources": True})
    for s in company.sources:
        if s.sourceType == "FS_DANOVE_SUBJEKTY":
            print(s.findings)
    await db.disconnect()

asyncio.run(main())
